Sparkle/el.lproj/Sparkle.strings

```diff
@@ -4,6 +4,8 @@
 
 "%@ %@ is currently the newest version available." = "Το %1$@ %2$@ είναι η τελευταία διαθέσιμη έκδοση.";
 
+"%@ %@ is currently the newest version available.\n(You are currently running version %@.)" = "Το %1$@ %2$@ είναι η τελευταία διαθέσιμη έκδοση.\n(Die derzeit installierte Version ist %3$@.)";
```

Comment:
The German translation slipped into the parenthes

Reply:
ouch. should i send a PR?